torch/distributed/_tensor/device_mesh.py

```diff
@@ -174,12 +174,15 @@ def __init__(
         # private field to pre-generate DeviceMesh's hash
         self._flatten_mesh_list = tuple(self.mesh.flatten().tolist())
         self._hash = hash((self._flatten_mesh_list, self.mesh.shape))
-        # always try to create default (world) pg, even if it is not initialized
-        # already. The world pg is used for device mesh identity (rank) on each
-        # process (we need to know if the current global rank is in the mesh or not)
-        self._get_or_create_default_group()
-        if _init_process_groups:
-            self._init_process_groups(_validate_mesh)
+
+        # Skip process group initialization if xla device.
+        if device_type != "xla":
```

Comment:
Let's also add a TODO here to remind us to make DeviceMesh have backend registration mechanism for XLA to integrate.

Reply:
Done